project_task_pull_request/__openerp__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Specialty Medical Drugstore
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Project Task Pull Request",
+    "summary": "Adds a field for a PR URI to project tasks",
+    "version": "9.0.1.0.0",
+    "category": "Project Management",
+    "website": "https://odoo-community.org/",
+    "author": "SMDrugstore, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
```

Comment:
can be removed if False as it's the default value

Reply:
Done